CRM/Usermenu/Upgrader/Setups/AddNavigationItems.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * User Menu Add Navigation Items setup.
+ */
+class CRM_Usermenu_Upgrader_Setups_AddNavigationItems {
+
+  /**
+   * Adds the base navigation items required by the user menu extension.
+   *
+   * @return Bool
+   */
+  public function apply() {
+    $userMenuValues = [
+      'label' => 'User Menu',
+      'name' => 'user-menu-ext__user-menu',
+      'icon' => 'fa fa-user',
+      'url' => '',
+      'permission' => 'access CiviCRM',
+      'is_active' => 1,
+    ];
+    $homeMenuValues = [
+      'label' => 'Home',
+      'name' => 'user-menu-ext__home-menu',
+      'url' => '/civicrm/dashboard?reset=1',
+      'permission' => 'access CiviCRM',
+      'is_active' => 1,
+    ];
+    $userMenuChildItemsValues = [
+      [
+        'label' => 'CiviCRM Home',
+        'name' => 'user-menu-ext__user-menu__civicrm-home',
+        'icon' => 'fa fa-home',
+        'url' => '/civicrm/dashboard?reset=1',
+        'is_active' => 1,
+      ],
+      [
+        'label' => 'My Account',
+        'name' => 'user-menu-ext__user-menu__my-account',
+        'icon' => 'fa fa-edit',
+        'url' => '/user/',
```

Comment:
Was it decided that this will lead to the account page rather than the account edit page for the user?

Reply:
Yes, it has been confirmed that we should redirect to the Drupal's user account and not to the edit page.